emacs/merlin.el

```diff
@@ -611,6 +611,34 @@ argument (lookup appropriate binary, setup logging, pass global settings)"
         (kill-local-variable 'merlin-buffer-configuration)
         (kill-local-variable 'merlin-erroneous-buffer)))))
 
+(defcustom merlin-stop-server-on-opam-switch t
```

Comment:
What is the point of this configuration option ? If the user decides to use the `opam-switch-mode` we probably always want to stop the server before switching.

Reply:
> we probably always want to stop the server before switching.

Exactly, this is what most users want, so it is the default option. But in some cases, the option might be useful (e.g., if a user starts two emacs instances, initially with the same opam switch, then want to change the switch in one instance… without killing the server for the other instance :)